.github/workflows/linting-backend.yml

```
@@ -0,0 +1,27 @@
+# https://deno.land/manual@v1.19.2/continuous_integration
+
+name: CI - Linting - Backend
+
+on:
+  pull_request:
+    paths:
+      - 'backend/**'
+  workflow_dispatch:
+
+jobs:
+  lint-backend:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
```

We should use the same version in all pipelines I think. This is the version from `update-api-client.yml` 😊 
```suggestion
          deno-version: vx.x.x
```